<?xml version="1.0"?>

<launch>
  <arg name="project" default="waypoint_driving"/>
  
  <!-- Parameters -->
  <rosparam file="$(find waypoint_driving)/config/params.yaml" command="load" /> 
  
  <!--- Start Node -->
  <node pkg="$(arg project)" type="driving_node" name="driving_node" output="screen" respawn="false"/>  
  
  <node pkg="$(arg project)" type="cloud_segment" name="cloud_segment" output="screen" respawn="false"/>
  
  <!--- Start Serial Node -->
  <node pkg="rosserial_python" type="serial_node.py" name="argos_mr_core" output="screen">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="115200"/>
  </node> 
</launch>
